client/js/lounge.js

```diff
@@ -1331,7 +1331,7 @@ $(function() {
 					} else {
 						var current = $(this);
 
-						if (current.attr("class") === "date-marker") {
+						if (current.hasClass("date-marker") && prev.hasClass("date-marker")) {
```

Comment:
The intended behaviour is to keep the lowest date marker, does this code not accomplish this?